scripts/actions/mobskills/occultation.lua

```diff
@@ -12,9 +12,13 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local base = math.random(10, 25)
+    local base = 10
 
-    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BLINK, base, 0, 120))
+    if mob:isNM() then
+        base = math.random(10, 25)
+    end
+
+    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BLINK, base, 0, 300))
```

Comment:
Jimmy's sheet says this effect, when stolen, retrieves utsusemi

```suggestion
    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.COPY_IMAGE, base, 0, 300))
```

the `return xi.effect.BLINK` will have the correct messaging, though I wonder what happens if it overwrites itself. Maybe we need a check for the message that returns from `xi.mobskills.mobBuffMove`

Reply:
Adjusted that to COPY_IMAGE, ended up needing to use the subpower to pass the shadow count for this one.